data/Duke.txt

```diff
@@ -0,0 +1,3 @@
+T | 0 | borrow book
+T | 0 | eat
+E | 0 | meeting|  May 1 2020
```

Comment:
Careful of the spacing over here! Depending on how your code runs, the spaces might affect the print of output!